README.md

```diff
@@ -125,8 +122,25 @@ terafoundation:
                 brokers: "localhost:9092"
 ```
 
+With an encrypted connection to kafka broker:
 
-
+```yaml
+terafoundation:
+    connectors:
+        kafka:
+            default:
+                brokers: "localhost:9092"
+                security_protocol: "ssl"
+                ssl_ca_location: "app/certs/CAs/rootCA.pem"
+                caCertificate: |         # if provided then ssl_ca_location will be ignored 
```

Comment:
Acknowledging that you address this in your comment, but I think it's a little confusing to provide an example with two mutually exclusive configuration settings.  I will accept anyway.